blazor/pager/accessibility.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Accessibility in Blazor Pager Component | Syncfusion
+description: Checkout here and learn about all Accessibility in Syncfusion Blazor Pager components and much more details.
```

Comment:
Checkout here and learn about accessibility in Syncfusion Blazor Pager component and much more details.